shared/actions/devices.js

```diff
@@ -103,11 +112,20 @@ export function removeDevice (deviceID: string, name: string, currentDevice: boo
       revokeRevokeDeviceRpc({
         param: {deviceID, force: false},
         callback: error => {
-          dispatch({
-            type: Constants.deviceRemoved,
-            payload: error,
-            error: !!error,
-          })
+          if (error) {
+            dispatch({
+              type: Constants.deviceRemoved,
+              payload: error,
+              error: true,
+            })
+          } else {
+            dispatch({
+              type: Constants.deviceRemoved,
+              payload: error,
+              error: false,
```

Comment:
ah yeah, and it's also typed as `Action` which is just `{type: any, payload: any ...}`